include/cetl/pf20/span.hpp

```diff
@@ -687,7 +689,7 @@ class span<T, dynamic_extent>
     ///
     /// Returns if the span has a zero size or not.
     /// @return true if the span size is 0.
-#if (__cplusplus >= CETL_CPP_STANDARD_17)
+#if (__cplusplus >= 201703L)
```

Comment:
link to the published C++ standards numbers would be useful, but I tried to find it myself and had difficulty. This might be the only definitive answer https://en.cppreference.com/w/cpp/preprocessor/replace